charts/logging-apps/values.yaml

```diff
@@ -41,7 +41,7 @@ filebeat:
 # fluentd -- [fluentd](https://github.com/fluent/fluentd) ([example](./examples/fluentd.yaml))
 # @default -- -
 fluentd:
-  # fluentd.enabled -- Enable loki
+  # fluentd.enabled -- Enable fluentd
```

Comment:
I guess fixing this typo was introduced by `helm-docs`?

Reply:
It was probably copy-pasta from when we added fluentd.